{"name": "test_02_warning", "status": "failed", "statusDetails": {"message": "AssertionError: assert False == True", "trace": "self = <Project.testcases.test_a8e.TestCase object at 0x00000238D5FB7EE0>\n\n    def test_02_warning(self):\n        test_qnx = qnx.qnx(\"1234567\",\"192.168.118.2\",\"root\",\"\")\n        test_qnx.qnx_screenshot(\"/var/share/\")\n        test = adb.adb.adb_pull(\"1234567\",\n                                \"/data/nfs/nfs_share/screenshot.bmp\",\n                                \"./testdata/actual/screenshot_warning.bmp\")\n        test_image=images.Images()\n        test_result = test_image.compare_by_matrix_in_same_area(\"./testdata/actual/screenshot_warning.bmp\",\n                                                                \"./testdata/expect/screenshot_warning.bmp\",\n                                                                (711,24,1227,80))\n>       assert test_result == True\nE       assert False == True\n\ntestcases\\test_a8e.py:39: AssertionError"}, "start": 1709354166082, "stop": 1709354171489, "uuid": "210e7fc4-ce2c-45fc-b7de-ad3522a1dde4", "historyId": "f39cfdab6574b10702827c550d90cb5b", "testCaseId": "f39cfdab6574b10702827c550d90cb5b", "fullName": "testcases.test_a8e.TestCase#test_02_warning", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_a8e"}, {"name": "subSuite", "value": "TestCase"}, {"name": "host", "value": "test-1"}, {"name": "thread", "value": "7688-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_a8e"}]}